external/common/include/cycle_counter.hpp

```diff
@@ -233,7 +233,13 @@ INLINE_ELAPSED(__inline__)
 #endif
 
 /* Visual C++ (FIXME: how to detect compilation for x86-64?) */
+/* Ref: Pre-defined C/C++ Compiler Macros http://sourceforge.net/p/predef/wiki/Home/ */
+// MSVC++ 8.0 _MSC_VER = 1400
 #if _MSC_VER >= 1400 && !defined(HAVE_TICK_COUNTER)
+#if defined(__amd64__) || defined(__x86_64__) || \
```

Comment:
>What is the problem this change is addressing? 

@scottpurdy This line was not present problems because it was not by GNU/CLang. However on MSVC this happens because ULONG64 is not defined.

> And perhaps there is a newer version of this file that we can include rather than hacking it ourselves?

There's a newer version of this file but if we use it then we must include its respective static libraries because this new version add a lot of new changes that are in conflict with the current compiled libraries.